lib/app/constants/routes.dart

```diff
@@ -0,0 +1,7 @@
+class Routes {
+  static const String news = '/news';
+  static const String campaigns = '/campaigns';
+  static const String profiles = '/profiles';
+  static const String mfa = '/mfa';
+  static const String apps = '/apps';
```

Comment:
I used `apps` because it was the requirement in #249 .
That's the reason why I used the naming for the routes also. As the routes are not visible in a flutter app, I think this is not important.

Reply:
> I used `apps` because it was the requirement in #249 . That's the reason why I used the naming for the routes also. 

Makes sense. I get the idea, but i think requirements like this (especially if they come from us) are also up for discussion if deemed non-optimal. In this case, I'd really prefer sticking to the UI wording (at least, as long as there is no reason why not to, such as avoiding ambiguity or such). Maybe @NikoHadouken can shed some light on the reason for this naming?

> As the routes are not visible in a flutter app, I think this is not important.

Depends on your point of view. In general, I totally agree. First and foremost, the app should work and look as expected. Nevertheless, non-functional requirements like (easy) maintainability and understandability of the code and developer experience in general are important, too. Code should be as easily legible and understandable as possible and especially self-explanatory. If we work on the project from the start, we all know what we mean by `app` and that it refers to the "tools". In the long run, this might be a source for confusion, e.g. for new employees/devs.